tmpl/devcontainerDockerfile.tmpl

```diff
@@ -0,0 +1,27 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/debian/.devcontainer/base.Dockerfile
+
+# [Choice] Debian version (use bullseye on local arm64/Apple Silicon): bullseye, buster
+ARG VARIANT="buster"
+FROM mcr.microsoft.com/vscode/devcontainers/base:0-\${VARIANT}
```

Comment:
I wonder if we really need to tie ourselves to vscode base image, or can we generate a generic image?